src/nnfusion/engine/device/cuda.cpp

```diff
@@ -50,14 +62,21 @@ CudaEngine::CudaEngine()
     // GPU specific graph passes
     g_passes->push_back(make_shared<KernelFusionPass>());
     g_passes->push_back(make_shared<KernelProfilingPass>());
+    g_passes->push_back(make_shared<PatternSubstitutionPass>());
     g_passes->push_back(make_shared<BlockFusionPass>());
+    g_passes->push_back(make_shared<RammerBaseKPass>());
```

Comment:
The RammerBaseKPass is only designed for OSDI artifacts evaluation and will be deleted in the official release.

Reply:
We may delete it later? Because this pr will remove interpreter, if deleting rammer pass here, artifacts evaluation may fail.

Reply:
Sure. 

And the artifacts evaluation is in the osdi20_artifact branch. I think we can modify other branches if it does not affect the artifact branch.

Reply:
I misunderstood, and thought this master branch was for artifacts evaluation. Just updated and deleted rammer pass.